### ISSUE TITLE
error TS2339: Property 'Server' does not exist on type 'typeof "http"'

### ISSUE THREAD

**Issue by leizongmin** [Microsoft/TypeScript#16998]
**TypeScript Version:**  2.4.1
**Code**

```ts
import * as http from 'http';

const s = new http.Server();
```

**Expected behavior:** No error

**Actual behavior:** Output error: **error TS2339: Property 'Server' does not exist on type 'typeof "http"'**

------

Other information:

**Node.js Version**: 6.11.0
**Package @types/node Version**: 8.0.8

**Comment by RyanCavanaugh** [comment]
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/6363
